src/GitHub.InlineReviews/ViewModels/CommentViewModel.cs

```diff
@@ -60,9 +61,17 @@ public class CommentViewModel : ReactiveObject, ICommentViewModel
             User = user;
             UpdatedAt = updatedAt;
 
+            var canDelete = this.WhenAnyValue(
+                x => x.EditState,
+                x => x == CommentEditState.None && user.Login.Equals(currentUser.Login));
+
+            canDelete.Subscribe(b => CanDelete = b);
```

Comment:
You could use [`ObservableAsPropertyHelper`](https://reactiveui.net/docs/guidelines/framework/prefer-oaph-over-properties) to make this a bit neater.